extensions/mssql/src/objectExplorerNodeProvider/fileSources.ts

```diff
@@ -196,16 +196,15 @@ export class HdfsFileSource implements IFileSource {
 		});
 	}
 
-	private readdir(path: string): Promise<IFile[]> {
+	private listStatus(path: string): Promise<IFile[]> {
```

Comment:
Again, I think it was clearer when using filesystem naming conventions. readdir is logically what we're doing here. listStatus is less obvious (to me).